app/app.go

```diff
@@ -404,7 +406,7 @@ func NewCertiKApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		slashing.NewAppModule(app.slashingKeeper, app.accountKeeper, app.stakingKeeper),
 		params.NewAppModule(),
 		staking.NewAppModule(app.stakingKeeper, app.accountKeeper, app.supplyKeeper, app.certKeeper),
-		mint.NewAppModule(app.mintKeeper),
+		cosmosMint.NewAppModule(app.mintKeeper.Keeper),
```

Comment:
why do we do this? just define those methods in our module and have it here. Our mint module is different from Cosmos

Reply:
I just realized this problem was there for a while. It's fixed.